_people/joerg_winkler.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Jörg Winkler
+category: Team
+position: Research Assistant
+avatar: /assets/images/people/joerg_winkler.png
+social_links:
+  email: j.winkler@fu-berlin.de
+  github: joergi-w
+  linkedin: winklerj
+  keybase: joergi
+---
+
+As a PhD candidate at Freie Universität Berlin and member of the graduate school IMPRS-BAC at the Max Planck 
+Institute for Molecular Genetics I work on algorithms for the analysis of long non-coding RNA.
+I hold a Master of Science degree in Bioinformatics from University of Hamburg.
+
+I have developed the two applications *LaRA* for rapid structural RNA alignment and *MaRs* for RNA motif search.
```

Comment:
How can I insert a link here to the already existing LaRA page?

Reply:
Can you rebase on the master? The LaRA page is not on this branch yet.
Regarding the linkage, we should find some easy way to also link people with apps, but I am no expert :)

Reply:
Interesting Idea, we could automatically link-back / list all apps that an author has.